<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OperationDialog</class>
 <widget class="QDialog" name="OperationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>File Operation [Experimental]</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBoxTarget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>Target</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QRadioButton" name="radioOK">
           <property name="text">
            <string>OK</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroupTarget</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioBad">
           <property name="text">
            <string>Bad</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroupTarget</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioAll">
           <property name="text">
            <string>All</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroupTarget</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioSelection">
           <property name="text">
            <string>Selection</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroupTarget</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Destination:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLineEdit" name="txtDest">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="btnBrowse">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QGroupBox" name="groupBoxOp">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Op.</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="radioCopy">
           <property name="text">
            <string>Copy</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroupOp</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioMove">
           <property name="text">
            <string>Move</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroupOp</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioDelete">
           <property name="text">
            <string>Delete</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroupOp</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QGroupBox" name="overwriteGroupBox">
        <property name="title">
         <string>Overwrite</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QRadioButton" name="radioAskOverwrite">
           <property name="text">
            <string>Ask</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroupOverwrite</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioYesOverwrite">
           <property name="text">
            <string>Yes</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroupOverwrite</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioNoOverwrite">
           <property name="text">
            <string>No</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroupOverwrite</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QTreeWidget" name="treeWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Filename</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Destination</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Result</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnStart">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="statusLabel">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="indent">
      <number>8</number>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="autoScrollCheckBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Auto scroll</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>radioOK</tabstop>
  <tabstop>radioBad</tabstop>
  <tabstop>radioAll</tabstop>
  <tabstop>radioSelection</tabstop>
  <tabstop>radioCopy</tabstop>
  <tabstop>radioMove</tabstop>
  <tabstop>radioDelete</tabstop>
  <tabstop>txtDest</tabstop>
  <tabstop>btnBrowse</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroupTarget"/>
  <buttongroup name="buttonGroupOp"/>
  <buttongroup name="buttonGroupOverwrite"/>
 </buttongroups>
</ui>
